core/src/solve.jl

```diff
@@ -126,6 +126,7 @@ of Vectors or Arrow Primitives, and is added to avoid type instabilities.
 """
 struct TabulatedRatingCurve{C} <: AbstractParameterNode
     node_id::Vector{Int}
+    active::BitVector
```

Comment:
Shall we simply use `Vector{Bool}` here? It uses more memory, but should be faster to read and write, since bits don't need to be shuffled around.

Reply:
I was talking with Bart about this, the reason I suggested `BitVector` is that he was using a function that returned a `BitVector`, so no conversion step was necessary that way.

But if you think `Vector{Bool}` is faster, we should probably switch.

Reply:
Ok let's keep it the same. If it causes issues or shows up in profiling we can switch later.

Apparently the performance is not so clear cut: https://discourse.julialang.org/t/bitvector-vs-vector-bool-as-default-on-comparison-operations/49431